packages/typespec-client-generator-core/test/decorators.test.ts

```diff
@@ -2969,4 +2969,134 @@ describe("typespec-client-generator-core: decorators", () => {
       ok(testModel);
     });
   });
+
+  describe("scope decorator", () => {
```

Comment:
you may also test if the models used in operation have the correct usage.

Reply:
Actually, the model does not exist in the output package, test updated.